### DISCUSSION TITLE
The recent vulnerability?

### DISCUSSION THREAD

**Discussion by tiksa** [yuezk/GlobalProtect-openconnect#349] [General]
Hello,

Is the recently found GlobalProtect vulnerability relevant also with this open-source version?

https://www.paloaltonetworks.com/blog/2024/04/more-on-the-pan-os-cve/

Kind regards
Timo

**Comment by yuezk** [comment]
Hi @tiksa,

I read about the vulnerability you mentioned. This vulnerability affects the PAN-OS server-side software and does not impact this open-source client. I'm pretty sure about that.

**Comment by tiksa** [comment]
Ok thanks! Closing for now.
